app/gui/src/project-view/components/visualizations/TableVisualisationTooltip.ts

```diff
@@ -41,20 +40,25 @@ export class TableVisualisationTooltip implements ITooltipComp {
       return `<div style="display: inline-block; width: 10px; height: 10px; border-radius: 50%; background-color: ${color}; margin-left: 5px;"></div>`
     }
 
-    const dataQualityTemplate = `
-            <div style="display: ${getDisplay(params.numberOfNothing)};">
-                Nulls/Nothing: ${getPercentage(params.numberOfNothing)}% ${createIndicator(+getPercentage(params.numberOfWhitespace))}
-            </div>
-            <div style="display: ${getDisplay(params.numberOfWhitespace)};">
-                Trailing/Leading Whitespace: ${getPercentage(params.numberOfWhitespace)}% ${createIndicator(+getPercentage(params.numberOfWhitespace))}
-            </div>
-        `
+    const getDataQualityTemplate = () => {
+      let template = ''
+      params.dataQualityMetrics.forEach((obj) => {
+        const key = Object.keys(obj)[0]
+        const value = key ? obj[key] : null
+        if (key && value) {
+          const metricTemplate = `<div style="display: ${getDisplay(value)};">
+          ${key}: ${getPercentage(value)}% ${createIndicator(+getPercentage(value))}
+      </div>`
+          template = template + metricTemplate
+        }
+      })
+      return template
+    }
 
     this.eGui.innerHTML = `
             <div><b>Column value type:</b> ${params.value}</div>
             <div style="display: ${params.showDataQuality ? 'block' : 'none'};"">
-                <b>Data Quality Indicators</b>
```

Comment:
Removed intentionally?

Reply:
Yes - we didn't see the point having the header in the tooltip.